### PR TITLE
Disable tab switching for add channel button

### PR DESCRIPTION
**Pull request by swoolcock**
Make sure the "add channel" button cannot be tab switched to.

Updates framework to include ppy/osu-framework#1363

### CODE REVIEW

File: osu.Game.Tests/Visual/TestCaseTabControl.cs [hunk 1]
```diff
@@ -37,6 +47,21 @@ public TestCaseTabControl()
             {
                 text.Text = "Currently Selected: " + newFilter.ToString();
             };
+
+            AddStep("set selection to first tab", () => filter.Current.Value = filter.VisibleItems.First());
```
Comment (FreezyLemon) [inline osu.Game.Tests/Visual/TestCaseTabControl.cs]: Nitpick, but could you make these shorter? The last ones don't even fit on the buttons, and you could really just put "select first", "select next" etc.

Comment (swoolcock) [reply osu.Game.Tests/Visual/TestCaseTabControl.cs]: I guess?

**Comment by peppy** [comment]
Any reasons why tests are in osu! project and not framework? They seem to be testing basic functionality of a framework control.

**Comment by swoolcock** [comment]
Fair point, I just saw that a visual test already existed in the osu! project for `TabControl`, so I piggybacked off that.

**Comment by swoolcock** [comment]
Visual tests removed, minor change force pushed.

**Review by peppy — APPROVED** [review]

**Pull request merged**